lib/src/base-styles/_nx-color-swatches.scss

```diff
@@ -81,8 +136,29 @@
   --nx-swatch-pink-95: hsl(var(--nx-swatch-pink-hs), 95%);
   --nx-swatch-pink-97: hsl(var(--nx-swatch-pink-hs), 97%);
 
+  // Red
+  --nx-swatch-red-hs:0, 100%;
```

Comment:
Saturation here is supposed to be 80%

Reply:
> Saturation here is supposed to be 80%

Good catch. FIxed here: [c86d8c2](https://github.com/sonatype/sonatype-react-shared-components/pull/1197/commits/c86d8c2fc0500daf71260954824476501fda904a)